### PR TITLE
Write a healthcheck to ensure the latest 100 items are in ElasticSearch

### PR DESCRIPTION
**Pull request by matthew-andrews**
Well, this was fun.

### CODE REVIEW

File: server/controllers/item.js [hunk 1]
```diff
@@ -95,16 +94,6 @@ function updateItem(req) {
 								});
 						}
 					});
-			})
-			.then(function(res) {
```
Comment (matthew-andrews) [inline server/controllers/item.js]: this fixes a bug

Comment (matthew-andrews) [inline server/controllers/item.js]: this code is repeated a few lines above

File: server/controllers/item.js [hunk 2]
```diff
@@ -113,24 +102,16 @@ function updateItem(req) {
...
-		var url = capiUrls.getPath(opts.index, opts.id);
 
-		request(url, function (err, res, body) {
-			if (err) {
-				reject(err);
-			} else if (res.statusCode !== 200) {
-				reject('Capi request failed');
+	// Get a correct v1/v2 path depending on the index you are updateing
+	var url = capiUrls.getPath(opts.index, opts.id);
+	return fetch(url)
```
Comment (matthew-andrews) [inline server/controllers/item.js]: one less use of `request` :smile:

File: server/checks/latest-news.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+			.then(uuids => {
+
+				// See if we have them in our Elasticsearch
+				return signedFetch(`https://${CLUSTER}/v${index}_api_v2/_mget`, {
+						method: 'POST',
+						headers: {
+							'Content-Type': 'application/json'
+						},
+						body: JSON.stringify({
+							docs: Array.from(uuids).map(uuid => { return { type: 'item', _id: uuid, _source: false }; })
```
Comment (i-like-robots) [inline server/checks/latest-news.js]: darn ambiguous braces!

File: server/checks/latest-news.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+									.then(res => {
+										return { _id: uuid, found: res.ok };
+									});
+							}))
+
+							// Filter out the ones that can't be retrieved from CAPI
+							.then(capiResults => {
+								return new Set(capiResults
+									.filter(item => { return item.found; })
+									.map(item => { return item._id; }));
```
Comment (i-like-robots) [inline server/checks/latest-news.js]: `.map(item => item._id)` \0/

**Pull request merged**